image/base/install-oxipng

```diff
@@ -12,7 +12,11 @@ case "${dpkgArch##*-}" in
 esac
 
 # Install other deps
-apt -y -q install advancecomp jhead jpegoptim libjpeg-turbo-progs optipng
+apt -y -q install advancecomp jpegoptim libjpeg-turbo-progs
```

Comment:
Not really related to this PR but I did wonder why we are installing various jpeg packages in the `install-oxipng` script which I would think should only deal with `oxipng`.